ArduCopter/Parameters.cpp

```diff
@@ -1025,6 +1030,12 @@ ParametersG2::ParametersG2(void)
 #if AUTOTUNE_ENABLED == ENABLED
     ,autotune_ptr(&copter.autotune)
 #endif
+#if FRAME_CONFIG == HELI_FRAME
+    #if MODE_AUTOROTATE_ENABLED == ENABLED
+    ,arot_ptr(copter.arot)
+    ,helispdhgtctrl_ptr(copter.helispdhgtctrl)
+    #endif
+#endif
```

Comment:
@Gone4Dirt you forgot to put the "&" in front of the argument to pass the address.  
,arot_ptr(&copter.arot)

Reply:
done